cloud/team/team.go

```diff
@@ -0,0 +1,708 @@
+	}
+	if workspace == "" {
+		workspace = ctx.Workspace
+	}
+	teams, err := GetWorkspaceTeams(client, workspace, teamPagnationLimit)
+	if err != nil {
+		return err
+	}
+	if len(teams) == 0 {
+		return ErrNoTeamsFoundInWorkspace
+	}
+	var team astrocore.Team
+	if id == "" {
+		team, err = selectTeam(teams)
+		if err != nil {
+			return err
+		}
+	} else {
+		for i := range teams {
+			if teams[i].Id == id {
+				team = teams[i]
+			}
+		}
+		if team.Id == "" {
+			return ErrTeamNotFound
+		}
+	}
+	teamID := team.Id
+	resp, err := client.DeleteWorkspaceTeamWithResponse(httpContext.Background(), ctx.OrganizationShortName, ctx.Workspace, teamID)
+	if err != nil {
+		return err
+	}
+	err = astrocore.NormalizeAPIError(resp.HTTPResponse, resp.Body)
+	if err != nil {
+		return err
+	}
+	fmt.Fprintf(out, "Astro Team %s was successfully removed from workspace %s\n", team.Name, ctx.Workspace)
+	return nil
+}
+
+func selectTeam(teams []astrocore.Team) (astrocore.Team, error) {
+	table := printutil.Table{
+		Padding:        []int{30, 50, 10, 50, 10, 10, 10},
+		DynamicPadding: true,
+		Header:         []string{"#", "TEAMNAME", "ID"},
+	}
+
+	fmt.Println("\nPlease select the team you would like to update:")
+
+	teamMap := map[string]astrocore.Team{}
+	for i := range teams {
+		index := i + 1
+		table.AddRow([]string{
+			strconv.Itoa(index),
+			teams[i].Name,
+			teams[i].Id,
+		}, false)
+		teamMap[strconv.Itoa(index)] = teams[i]
+	}
+
+	table.Print(os.Stdout)
+	choice := input.Text("\n> ")
+	selected, ok := teamMap[choice]
+	if !ok {
+		return astrocore.Team{}, errInvalidTeamKey
+	}
+	return selected, nil
+}
+
+func GetWorkspaceTeams(client astrocore.CoreClient, workspace string, limit int) ([]astrocore.Team, error) {
+	offset := 0
+	var teams []astrocore.Team
+
+	ctx, err := context.GetCurrentContext()
+	if err != nil {
+		return nil, err
+	}
+	if ctx.OrganizationShortName == "" {
+		return nil, ErrNoShortName
+	}
+	if workspace == "" {
+		workspace = ctx.Workspace
+	}
+	for {
+		resp, err := client.ListWorkspaceTeamsWithResponse(httpContext.Background(), ctx.OrganizationShortName, workspace, &astrocore.ListWorkspaceTeamsParams{
+			Offset: &offset,
+			Limit:  &limit,
+		})
+		if err != nil {
+			return nil, err
+		}
+		err = astrocore.NormalizeAPIError(resp.HTTPResponse, resp.Body)
+		if err != nil {
+			return nil, err
+		}
+		teams = append(teams, resp.JSON200.Teams...)
+
+		if resp.JSON200.TotalCount <= offset {
+			break
+		}
+
+		offset += limit
+	}
+
+	return teams, nil
+}
+
+// Prints a list of all of an organizations teams
+func ListWorkspaceTeams(out io.Writer, client astrocore.CoreClient, workspace string) error {
+	ctx, err := context.GetCurrentContext()
+	if err != nil {
+		return err
+	}
+	if workspace == "" {
+		workspace = ctx.Workspace
+	}
+	table := printutil.Table{
+		Padding:        []int{10, 50, 50, 10, 10, 10, 10},
+		DynamicPadding: true,
+		Header:         []string{"ID", "Role", "Name", "Description", "CREATE DATE"},
```

Comment:
```suggestion
		Header:         []string{"ID", "Role", "Name", "Description", "Create Date"},
```
nit: just keeping the header consistent